en/12_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -66,13 +65,11 @@ While anyone in the community can review a pull request, only reviews from maint
   - If another reviewer feels confident, they can offer to review it instead
 - Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
   - If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
-- After two maintainers have approved the pull request, merge it
-  - The person who merges the PR can be one of the two approving reviewers
-  - The person who merges the PR must also understand and approve the changes
-  - The person who contributed the PR must not be included as one of the two approving reviewers
-- If you don't feel confident clicking merge, ask for a second opinion. Don’t approve the pull request if you wouldn’t feel confident merging it - instead, say where you got to with it (e.g. tested, worked well locally, couldn’t find any problems but want another opinion) and get another reviewer to check it
+- Make sure Core Committers and CMS Squad are aware when a pull request has been approved and is ready to be merged
+- Don’t approve the pull request if you wouldn’t feel confident merging it - instead, say where you got to with it (e.g. tested, worked well locally, couldn’t find any problems but want another opinion) and get another reviewer to check it
   - If no other reviewer feels confident about it, but it seems like it’s probably a good change in general, escalate to Core Committers
 - If at any point there is a disagreement between reviewers that seems unresolvable, escalate the discussion to Core Committers.
+- If any pull requests seem to be fixing a security issue, immediately notify the CMS Squad and Core Committers
```

Comment:
This last note was in the "Contribution refiners" section but for some reason wasn't here.